[docs] Update the documentation to include a description of the framework's window management capabilities (#5363)
DevExpress/testcafe: docs/articles/documentation/reference/test-api/testcontroller/closewindow.md

---
layout: docs
title: t.closeWindow Method
permalink: /documentation/reference/test-api/testcontroller/closewindow.html
---

# t.closeWindow method

Closes a browser window.

```js
t.closeWindow( [windowDescriptor] )
```

Parameter | Type | Description
--------- | ---- | ------------

> windowDescriptor *(optional)* | Object | Object that describes the target window. If this parameter is omitted, the currently active window is selected.

Reviewer: What is the difference between an 'active' window and a 'currently active' window?

> >You can not close the last remaining window, or windows with open children.

Reviewer 2: ```suggestion
>You cannot close the last remaining window or windows with open children.
```

**Examples:**

> The following two examples produce identical test scenarios.

Reviewer: ```suggestion
The following two examples produce identical test scenarios:
```

> The `window` parameter is absent. Each call of the `t.closeWindow` method closes the currently active window:

Reviewer: Same here
